### ISSUE TITLE
dynamically rendered components are not registered for Unicorn.call()

### ISSUE THREAD

**Issue by martindemello** [adamghill/django-unicorn#437]
`Unicorn.call()` does not appear to find components that are loaded dynamically inside an `if` block in the template. I've uploaded a full demo to https://github.com/martindemello/test_unicorn, but the relevant bit is:

main:
```
    {% unicorn 'parent' %}
    <script src="{% static 'foo/main.js' %}"></script>
```
script:
```
function call_child(target) {
  Unicorn.call(target, 'hello')
}
```

parent:
```
class ParentView(UnicornView):
    active_view = None

    def toggle(self):
        self.active_view = 'child1' if self.active_view == 'child2' else 'child2'
```
```
  <button unicorn:click='toggle'>toggle</button>
  <div>
    {% if active_view %}
    {% unicorn active_view key=active_view %}
    {% endif %}
  </div>
```

children:
```
class Child1View(UnicornView):
    def hello(self):
        print('called on child1')
```
```
<div>
  this is child1!
  <button onclick="call_child('child1')">call</button>
</div>
```

**Comment by pietrodicaprio** [comment]
Same issue here, any plan to have dynamically loaded components to work properly?

**Comment by adamghill** [comment]
I think this should be fixed by 0.58.0 of `Unicorn` which drastically improves child components so I'm going to close it. If it is still a problem feel free to re-open this and create a failing test or small example in the `example` app of this repo to replicate, though. Thank you!
